### PR TITLE
fix: issues with npm run start

### PR DESCRIPTION
**Pull request by nmerget**

### CODE REVIEW

File: showcases/angular-showcase/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "@angular/platform-browser": "~13.3.12",
     "@angular/platform-browser-dynamic": "~13.3.12",
     "@angular/router": "~13.3.12",
-    "@db-ui/foundations": "*",
+    "@db-ui/foundations": "file:../../packagesi/foundations",
```
Comment (mfranzke) [inline showcases/angular-showcase/package.json:22]: Is there  a typo in the path ("packagesi"), and in the other files as well ?

**Review by mfranzke — CHANGES_REQUESTED** [review]

File: showcases/react-showcase/package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@db-ui/foundations": "*",
+    "@db-ui/foundations": "file:../../packagesi/foundations",
```
Comment (mfranzke) [inline showcases/react-showcase/package.json:16]: @nmerget Is there a typo in the path ("packagesi"), and in the other files as well ?
```suggestion
    "@db-ui/foundations": "file:../../packages/foundations",
```

File: showcases/svelte-showcase/package.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@db-ui/foundations": "file:./../../packages/foundations"
+    "@db-ui/foundations": "file:../../packagesi/foundations"
```
Comment (mfranzke) [inline showcases/svelte-showcase/package.json:13]: @nmerget Is there a typo in the path ("packages**i**")?
```suggestion
    "@db-ui/foundations": "file:../../packages/foundations"
```

File: showcases/vanilla-showcase/package.json [hunk 4]
```diff
@@ -15,7 +15,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@db-ui/foundations": "file:./../../packages/foundations"
+    "@db-ui/foundations": "file:../../packagesi/foundations"
```
Comment (mfranzke) [inline showcases/vanilla-showcase/package.json:18]: @nmerget Is there a typo in the path ("packages**i**")?
```suggestion
    "@db-ui/foundations": "file:../../packages/foundations"
```

File: showcases/vue-showcase/package.json [hunk 5]
```diff
@@ -12,7 +12,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@db-ui/foundations": "*",
+    "@db-ui/foundations": "file:../../packagesi/foundations",
```
Comment (mfranzke) [inline showcases/vue-showcase/package.json:15]: @nmerget Is there a typo in the path ("packages**i**")?
```suggestion
    "@db-ui/foundations": "file:../../packages/foundations",
```

File: showcases/next-showcase/package.json [hunk 6]
```diff
@@ -13,7 +13,7 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@db-ui/foundations": "file:./../../packages/foundations",
+    "@db-ui/foundations": "file:../../packagesi/foundations",
```
Comment (mfranzke) [inline showcases/next-showcase/package.json:16]: @nmerget Is there a typo in the path ("packages**i**")?
```suggestion
    "@db-ui/foundations": "file:../../packages/foundations",
```

File: showcases/angular-showcase/package.json (hunk 1 above)
Comment (mfranzke) [inline showcases/angular-showcase/package.json:22]: @nmerget Is there a typo in the path ("packages**i**")?
```suggestion
    "@db-ui/foundations": "file:../../packages/foundations",
```

File: showcases/react-showcase/package.json (hunk 2 above)
Comment (nmerget) [reply showcases/react-showcase/package.json]: fixed

File: showcases/next-showcase/package.json (hunk 6 above)
Comment (nmerget) [reply showcases/next-showcase/package.json]: fixed

File: showcases/angular-showcase/package.json (hunk 1 above)
Comment (nmerget) [reply showcases/angular-showcase/package.json]: fixed

File: showcases/vue-showcase/package.json (hunk 5 above)
Comment (nmerget) [reply showcases/vue-showcase/package.json]: fixed

File: showcases/svelte-showcase/package.json (hunk 3 above)
Comment (nmerget) [reply showcases/svelte-showcase/package.json]: fixed

**Review by mfranzke — APPROVED** [review]

**Pull request merged**